pslam-baseline.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This filename looks wrong. It is currently *pslam* when it should probably be *psalm*.

Reply:
Yeah, I made a typo when running command to generate it, but the file is still used :)
I'll update it.